spring-data-neo4j/pom.xml

```diff
@@ -194,6 +194,80 @@
             <artifactId>spring-webmvc</artifactId>
             <scope>test</scope>
         </dependency>
+		<dependency>
+			<groupId>org.jetbrains.kotlin</groupId>
+			<artifactId>kotlin-stdlib-jdk8</artifactId>
+			<version>${kotlin.version}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.jetbrains.kotlin</groupId>
+			<artifactId>kotlin-reflect</artifactId>
+			<version>${kotlin.version}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.jetbrains.kotlin</groupId>
+			<artifactId>kotlin-test</artifactId>
+			<version>${kotlin.version}</version>
+			<scope>test</scope>
+		</dependency>
+
+	</dependencies>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.jetbrains.kotlin</groupId>
+				<artifactId>kotlin-maven-plugin</artifactId>
```

Comment:
1. Do we now compile everything (also java) by the kotlin compiler?
2. Is that safe? 
3. Is this how other SD projects do it?

Reply:
I just applied the suggestions from Intellij. Looks a lot liek what's in the kotlin official docs.
I presume (maybe wrong) kotlin does not reimplement a full java compiler but delegates to javac.